src/Nav.js

```diff
@@ -53,6 +68,7 @@ function Nav({
 
       <div className="hidden md:flex items-center justify-end space-x-8 md:flex-1 lg:w-0">
         <Input
+          id="searchBar"
```

Comment:
by convention, classes ids, and attributes are kebab case (search-bar)
```suggestion
          id="search-bar"
```

Reply:
I was unaware of that. Thank you for letting me know!